deploy/package.sh

```diff
@@ -25,6 +27,10 @@ DEPLOY_PGSQL=${13:-${DEPLOY_PGSQL:-"true"}}
 
 set -ex
 
+if [[ "$AI_MODEL_INFRA_TYPE" != "batch-account" ]] && [[ "$AI_MODEL_INFRA_TYPE" != "aks" ]]; then
```

Comment:
Same feedback. Let's not include this flag in this PR

Reply:
Removed this reference from AKS PR.